src/py/aspen/covidhub_import/implementation.py

```diff
@@ -97,24 +126,85 @@ def import_project(
     with session_scope(interface) as session:
         project = (
             covidhub_session.query(Project)
+            .options(
+                joinedload(
+                    Project.covidtracker_group, covidtracker.GroupToProjects.group
+                )
+            )
             .filter(Project.rr_project_id == rr_project_id)
             .one()
         )
 
-        group = (
-            session.query(Group)
-            .filter(Group.name == project.originating_lab)
-            .one_or_none()
-        )
-        if group is None:
-            # copy the project info into the "group"
-            group = Group(
-                name=project.originating_lab,
-                address=project.originating_address,
+        group, admin_group, cdph_group = (
+            get_or_make_group(
+                session,
+                project.originating_lab,
+                project.originating_address,
+                # FIXME: this needs to be updated.
+                email="fakeemail@dph.gov",
+            ),
+            get_or_make_group(
+                session,
+                "ADMIN",
+                "123 Main St.",
                 # FIXME: this needs to be updated.
                 email="FAKE_EMAIL@gmail.com",
+            ),
+            get_or_make_group(
+                session,
+                "CDPH",
+                "3701 N Freeway Blvd, Sacramento, CA 95834",
+                # FIXME: this needs to be updated.
+                email="cdph.internetadmin@cdph.ca.gov",
+            ),
+        )
+        covidhub_group_to_aspen_group = {
+            project.covidtracker_group.group: group,
+            covidhub_session.query(covidtracker.Group)
+            .filter(covidtracker.Group.name == "Admin")
+            .one(): admin_group,
+            covidhub_session.query(covidtracker.Group)
+            .filter(covidtracker.Group.name == "CDPH")
+            .one(): cdph_group,
+        }
+
+        covidhub_users: Iterable[covidtracker.UsersGroups] = covidhub_session.query(
+            covidtracker.UsersGroups
+        ).filter(
+            or_(
+                covidtracker.UsersGroups.group_id.in_(
+                    covidhub_session.query(covidtracker.Group.id).filter(
+                        covidtracker.Group.name.in_(["Admin", "CDPH"])
```

Comment:
same here? Or is this not case sensitive?

Reply:
Same here, in the old DB, it was Admin and CDPH.